projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -144,8 +128,10 @@ export function NoticeBox( props ) {
 			status={ 'is-info' }
 			className={ noticeBoxClassName }
 			onDismissClick={ dismissNoticeBox }
+			icon={ 'info-outline' }
 		>
-			{ notice.message }
+			{ notice.header && <span className="dops-notice__header">{ notice.header }</span> }
```

Comment:
If this is a heading it feels like we should mark it up as such.

In our page it's a `h3` ('Your search records' is h2).

Reply:
```suggestion
			{ notice.header && <h3 className="dops-notice__header">{ notice.header }</h3> }
```

Reply:
(This gets more interesting in reusable components - for example if we were adding a heading to SimpleNotice itself! Some discussion here: https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#headings-in-reusable-components)